shared/middleware/helpers/property_bag.rb

```diff
@@ -114,6 +116,12 @@ def set(name, value, ip_address, time = DateTime.now)
       },
     )
     value
+  rescue Aws::DynamoDB::Errors::ValidationException => e
+    if e.message == 'Item size has exceeded the maximum allowed size'
+      too_large
```

Comment:
That link is helpful. That seems like a reasonable approach to take for this. I'll look into going that route.